### PR TITLE
PYTHON-993:  Add host kwarg to Session.execute/execute_async APIs to send a query to a specific node

### PR DESCRIPTION
**Pull request by aboudreault**

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -9,6 +9,7 @@ Features
 * Allow encrypted private keys for 2-way SSL cluster connections (PYTHON-995)
 * Introduce new method ConsistencyLevel.is_serial (PYTHON-1067)
 * Add Session.get_execution_profile (PYTHON-932)
+* Add host kwarg to Session.execute/execute_async APIs to send a query to a specific node (PYTHON-933)
```
Comment (aboudreault) [inline CHANGELOG.rst]: will fix to PYTHON-993 before merging

**Comment by mambocab** [comment]
While I was running the tests locally, I wanted to add an assertion to the test to check that the LBP wasn't used so [I made a branch](https://github.com/datastax/python-driver/compare/python-933...mambocab:python-933_comments). Couple other small changes as well:

- grammar/wording edits to docstrings & comments
- changed test to use a random known host (allows us to remove `@local`, checks against several hosts instead of a fixed one)

Feel free to squash without attribution.

Working with the `Cluster`- and `Session`-level EP APIs was a bit annoying so I filed [https://datastax-oss.atlassian.net/browse/PYTHON-1071](https://datastax-oss.atlassian.net/browse/PYTHON-1071) for future followup.

**Comment by mambocab** [comment]
Also, +1 after merging those small edits!

**Comment by aboudreault** [comment]
Thanks for your review and changes. The test is way better like this, :+1:

**Pull request merged**